gensim/models/hdpmodel.py

```diff
@@ -525,6 +530,12 @@ def hdp_to_lda(self):
         beta = (self.m_lambda + self.m_eta) / (self.m_W * self.m_eta + \
                 self.m_lambda_sum[:, np.newaxis])
 
+        # if the user requests to return an ldamodel instead, it is returned with the appropriate alpha and beta values
+        if model:
+            ldam = ldamodel.LdaModel(num_topics=150, alpha=alpha, id2word=self.id2word)
```

Comment:
No, because the alpha returned here is for 150 topics. If we wish to do that then we have to change the shape of alpha.